### DISCUSSION TITLE
Make the vertical scrolling within the body only

### DISCUSSION THREAD

**Discussion by SourceCipher** [glideapps/glide-data-grid#775] [Ideas]
Is it possible to make the vertical scrolling within the actual list of items and not from the header?

![image](https://github.com/glideapps/glide-data-grid/assets/351018/7e9f65b6-dcc1-4469-853b-dffb78d19332)
![image](https://github.com/glideapps/glide-data-grid/assets/351018/5d2bccdc-64e0-4fdb-8a7f-aad61079a111)

**Comment by jassmith** [comment]
its more than likely you didn't constrain the height of the data editor. This is really bad for many reasons. The data editor should in general never be taller than the window size at most.
